lib/bb/generate_report_request_form.rb

```diff
@@ -7,7 +7,7 @@ class GenerateReportRequestForm < Common::Form
 
     ELIGIBLE_DATA_CLASSES = %w( seiactivityjournal seiallergies seidemographics
                                 familyhealthhistory seifoodjournal healthcareproviders healthinsurance
-                                seiimmunizations labsandtests medicalevents medications militaryhealthhistory
```

Comment:
It's being used in specs. I think it's helpful to have visual queue too to what the available types are, but you're right in assuming that it's not used in the main code. We fetch them for each user.